app/src/app/components/view-profile/view-profile.component.scss

```diff
@@ -0,0 +1,173 @@
+adf-view-profile {
+  letter-spacing: .5px;
+
+  .profile_container {
+    margin-top: 1rem;
+    overflow: scroll;
+    height:100%;
+    width: 100%;
+  }
+
+  .profile_row {
+    width: 100%;
+  }
+
+  .profile_title {
+    display: flex;
+    margin-left: 2rem;
+  }
+
+  .profile {
+    cursor: pointer;
+  }
+
+  .profile_general_row {
+    margin: 2rem 0 0 2rem;
+    width: 70%;
+    border: 1px solid var(--theme-grey-background-color);
+    border-radius: 1rem;
+  }
+
+  .profile_icon{
+    margin-right: 1rem;
+    margin-top: 1rem;
+    cursor: pointer;
+  }
+
+  .profile_general_icon {
+    cursor: pointer;
+  }
+
+  .profile_text {
+    letter-spacing: .5px;
+  }
+
+  .profile_general {
+    display: flex;
+    padding-left: 1rem;
+  }
+
+  .profile_general_bottom_radius {
+    border-bottom-left-radius:0;
+    border-bottom-right-radius: 0;
+  }
+
+  .profile_general_div {
+    display: flex;
+    width: 60%;
+    padding-top: 1rem;
+    cursor: pointer;
+  }
+
+  .general_title {
+    margin-left: 0.6rem;
+    margin-top: 4px;
+    letter-spacing: .5px;
+  }
+
+  .general_edit_btn {
+    width:60%;
+    text-align: end;
+  }
+
+  .mat_divider {
+    border-top-width: 2px;
+    border-top-color: var(--theme-grey-background-color);
+    padding-left: -1px;
+    padding-right: 1px;
+  }
+
+  .profile_row {
+    width: 100%;
+    margin: 2rem 0 0 0;
+  }
+
+  .general_edit {
+    margin: 0.7rem;
+    background: var(--theme-grey-text-background-color);
+  }
+
+  .general_edit:hover {
+    background: var(--theme-grey-hover-background-color);
+  }
+
+  .general_cancel_btn {
+    margin: 0.7rem;
+    background-color: var(--theme-grey-text-background-color);
+  }
+
+  .general_save_btn {
+    margin: 0.7rem;
+    color: white;
+    background-color: var(--theme-blue-button-color);
+  }
+
+  .general_dropdown_divider {
```

Comment:
I think you don't need this class here, and it can be removed from all dividers in the HTML, because at line 73 you already apply this color to all `.mat-divider` classes

Reply:
This class is used because the line fields the rows are of darker color than at the border. So different colors are used.